x-pack/plugins/security_solution/public/common/components/events_viewer/event_details_flyout.tsx

```diff
@@ -34,35 +37,42 @@ const EventDetailsFlyoutComponent: React.FC<EventDetailsFlyoutProps> = ({
   browserFields,
   docValueFields,
   timelineId,
+  handleCloseExpandedEvent,
 }) => {
-  const dispatch = useDispatch();
   const expandedEvent = useDeepEqualSelector(
     (state) => state.timeline.timelineById[timelineId]?.expandedEvent ?? emptyExpandedEvent
   );
 
-  const handleClearSelection = useCallback(() => {
-    dispatch(
-      timelineActions.toggleExpandedEvent({
-        timelineId,
-        event: emptyExpandedEvent,
-      })
-    );
-  }, [dispatch, timelineId]);
+  const [loading, detailsData] = useTimelineEventsDetails({
+    docValueFields,
+    indexName: expandedEvent.indexName!,
+    eventId: expandedEvent.eventId!,
+    skip: !expandedEvent.eventId,
+  });
+
+  const isAlert = useMemo(() => {
+    if (detailsData) {
+      return findIndex({ category: 'signal', field: 'signal.rule.id' }, detailsData) >= 0;
```

Comment:
maybe instead of `findIndex` we could use https://lodash.com/docs/4.17.15#some ?